.travis.yml

```diff
@@ -19,3 +19,5 @@ before_install:
 script:
 - docker run -it -v ~/id_rsa_worker:/root/.ssh/id_rsa_worker validation_worker
   bash -c "source test/e2e-test_positive_verification.sh"
+- docker run -it -v ~/id_rsa_worker:/root/.ssh/id_rsa_worker validation_worker
```

Comment:
This looks like duplicate code... Lines 20 and 21 are the same as 22 and 23.

Reply:
Also Lines 24 and 25...

Reply:
You are right, instead of spinning up each test separately, I could have run all tests in the same docker. But then I would have to reset env variables and ... That is the reason why I am starting a new docker image each time.

Reply:
IF I would understand Felix magic, I would apply this for sure:
https://github.com/gnosis/dex-zksnarks/blob/master/ZoKrates/test/run_tests.sh#L3